docs-kits/kits/Business Partner Kit/Documentation BPDM/bpdm_arc42.md

```diff
@@ -224,92 +216,8 @@ Due to a transmission phase there are two concepts of Business Partner Upload Mo
 
 -Currently there is no SME Application available
```

Comment:
Can you please add space here between '-' and 'Currently there is no SME Application available'